wpengine/lostromos
# Add docker image to build process.

Signed-off-by: Derek Rushing <derek.rushing21@gmail.com>

# What Are We Doing Here

We are adding a build/deploy of a docker image containing the lostromos binary.

## How to Verify

1. Check out this PR and run `make GOOS=linux build` and `make docker-build`, then use the docker image locally to interact with lostromos.
